pymatgen/ext/matproj.py

```diff
@@ -465,20 +466,20 @@ def get_entries(self, chemsys_formula_id_criteria, compatible_only=True,
             entries = sorted(entries, key=lambda entry: entry.data["e_above_hull"])
         return entries
 
-    def get_pourbaix_entries(self, chemsys):
+    def get_pourbaix_entries(self, chemsys, solid_compat=MaterialsProjectCompatibility()):
```

Comment:
@tschaume note the method signature changed, but the default should be fine

Reply:
@mkhorton yes that's part of the two minor changes I made in pymatpro to make the Pourbaix diagram at least show up again :P The other one is `energy` -> `_energy`. See https://github.com/materialsproject/pymatpro/commit/8aa0bac0779beb4d5504de4b80054add48dda447

Reply:
Thanks @tschaume! Thought this PR was mostly backwards-compatible, so my main interest (aside from fixing) is getting an appropriate test added to pymatgen so we know if we break anything the website depends on

Reply:
@tschaume @mkhorton I can look into this tonight / tomorrow morning. Can you give me more specifics about what the problem is and/or open an issue in django?

I agree the PR should have been backwards-compatible, but I know a lot of the stuff in pymatpro (esp. related to Pourbaix) is quite archaic. What version of pymatgen is the current version of the website running?

Reply:
No, this is fine. There's no way to make a test in pymatgen for the behavior in pymatpro. That is a dumpster of a code-base that I don't think we should be pinning behavior in pymatgen to. Rather pymatpro needs to be updated with respect to pymatgen. If there is a test for pymatpro working properly, that has to be in pymatpro.